pages/_profiles/Person/0.3-DRAFT.html

```diff
@@ -0,0 +1,310 @@
+---
+# spec_info content generated using GOWeb
+# DO NOT MANUALLY EDIT THE CONTENT
+cross_walk_url: https://docs.google.com/spreadsheets/d/1vliGiKWlHzqXWcGPa15l0PzJbQGXPKxYPq8Mups_or4
+dde_ui_url: https://discovery.biothings.io/view/bioschemasdrafts/bioschemasdrafts:Person
+gh_tasks: https://github.com/Bioschemas/specifications/labels/type%3A%20Person
+group: people
+hierarchy: []
```

Comment:
Hierarchy should look like
```
hierarchy:
- Thing
- Person
```
As it is empty right now, it is not parsed by the template 
![image](https://user-images.githubusercontent.com/5427252/206571106-c6280d91-b37b-4759-86c6-3306b610903b.png)